cookbook/case_studies/ml_training/spark_horovod/keras_spark_rossmann_estimator.py

```diff
@@ -150,10 +180,14 @@ def prepare_google_trend(
         ),
     )
 
-    # expand dates
+    # A call to expand dates function is returned.
     return expand_date(google_trend_all)
 
-
+# %%
+# Data pre-processing
+# ===================
+#
+# We define a function to expand the dates, i.e extract the year, month, week and day from a date.
```

Comment:
```suggestion
# Next, we set a few date-specific values in the DataFrame.
```

Reply:
I think we have a date, and we are extracting the year, month and week and day. not sure if we are setting the values..